tools/stern/src/Stern/Intra.hs

```diff
@@ -133,10 +133,10 @@ assertBackendApiVersion = recoverAll (constantDelay 1000000 <> limitRetries 5) $
     throwIO . ErrorCall $ "newest supported backend api version must be " <> show backendApiVersion
 
 path :: ByteString -> Request -> Request
-path = Bilge.path . ((toByteString' backendApiVersion <> "/") <>)
+path = Bilge.path . (("v" <> toByteString' backendApiVersion <> "/") <>)
```

Comment:
i think this must have changed under my nose?  or, more likely, i'm misremembering things.  oh well!